app/generators/contestantRoundScoreGenerator.tsx

```diff
@@ -1,27 +1,22 @@
 import { ITableRowData } from "../utils/wikiFetch"
-import { getTeamList, ITeam } from "../utils/wikiQuery"
+import { ITeam } from "../utils/wikiQuery"
+import { getNumberOfRounds } from '../utils/teamListUtils';
 import Team from '../models/Team'
 import LeagueStanding from '../models/LeagueStanding'
 
 interface Dictionary<T> {
     [Key: string]: T;
 }
 
-async function generateContestantRoundScores(dataFetcher: () => Promise<ITableRowData[]>, listOfContestantLeagueData: any[]) {
-
-    const wikiContestants = await dataFetcher()
-    const pageData = getTeamList(wikiContestants)
-
+export async function generateContestantRoundScores(dataFetcher: () => Promise<ITableRowData[]>, getCompetitorList: any, listOfContestantLeagueData: any[]) {
```

Comment:
Please add a better type for `getCompetitorList`

Reply:
Added type :)